Analysis/include/SampleDescriptor.h

```diff
@@ -20,15 +20,14 @@ struct AnalyzerSetup {
     Period period;
     SignalMode mode;
     QCDmethod qcd_method;
+    std::vector<std::string> qcd_shape;
```

Comment:
`qcd_shape` is not a vector, it is a single EventRegion.
```
EventRegion qcd_shape;
std::string qcd_shape_str;
```
Same for the qcd_shape